assets/data/community.json

```diff
@@ -388,10 +388,10 @@
     "currently_learning": "CSS"
   },
   {
-    "name": "NiclasTanskanen",
```

Comment:
Hi Anya,

There is a conflict with merging this pull request.
You will need to resolve this before we can merge your fork with the main repo.

If you have any questions feel free to reach out.
Thank you

Reply:
Hi, Okay I will have a look. Thank you

Reply:
@anyahush the problem isn't a merge conflict here, it's that you have removed "NiclasTanskanen"'s details from the community.json file and replaced them with your own 😄 I've attached a screenshot of the removal, and how it should look like.

What was removed:
![image](https://user-images.githubusercontent.com/48055438/197392044-ebeccca4-72e2-42e0-b583-73df03421bdb.png)

How it should be:
![image](https://user-images.githubusercontent.com/48055438/197392170-7f26bec8-51c4-494b-b379-f351b1c2026f.png)

Can you put those details back in please? and then add your own object following the guidelines in the [basic.md](https://github.com/auxfuse/hackathon-git-labs/blob/main/basic.md) file. 🤜 Everything else in this is great, we just need to make sure that the `community.json`  file is handled correctly.

When this is done, add, commit, and push back to the branch associated with this PR (in this case, you are on the main branch of your fork, we recommend using branches for subsequent PRs as per the documentation for this project) to auto-update this PR and let us know via comment. We will review, and then will resolve this conversation too pending approval & merging. 🎉

Reply:
So sorry about that!! Silly me 🙈 I have fixed the error. Thanks for your help 😊